dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/publish/servers/FaultTolerance20TCKServer/jvm.options

```diff
@@ -1 +1,2 @@
 -Dcom.ibm.tools.attach.enable=yes
+-Xjit:{org/eclipse/microprofile/config/ConfigProvider.getConfig(Ljava/lang/ClassLoader;)Lorg/eclipse/microprofile/config/Config;}(tracefull,traceInlining,traceCG,log=getConfig.trace)
```

Comment:
We don't want this change. This was just for when I was running personal builds to diagnose a problem on OpenJ9 12.